doc/README.md

```diff
@@ -471,6 +471,34 @@ currently selected entry is marked with a ✓. Clicking on the 🗙 on the
 right-hand side of such an entry deletes the configuration (after
 prompting the user to confirm).
 
+## Flame graph
+
+The `Flame graph (new)` view displays profile information as a
+[flame graph](https://www.brendangregg.com/flamegraphs.html).
+
+Boxes on this view corresponding to stack frames in the profile. Caller boxes
+are directly above callee boxes. The width of each box is proportional to the
+number of profile samples where that frame was present on the call stack.
```

Comment:
"number of profile samples" is a bit overfitting IMO since it implicitly assumes samples are unweighted. Earlier docs introduce the concept of "sample value", I would perhaps use it here. "The width of each box is proportional to the sum of the sample value of profile samples where that frame ...".